jenkins-pipelines/artifacts-ami-arm.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#! groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+artifactsPipeline(
+    test_config: 'test-cases/artifacts/ami.yaml',
+    backend: 'aws',
+    provision_type: 'spot_low_price',
+    availability_zone: 'c',  // Since ARM processors aren't available in zone 'a' (the default) at the moment.
```

Comment:
for all the regions?

Reply:
Yes. I went though all of the regions that support arm at all and zone c was always available